samples/multicore/hello_world/README.rst

```diff
@@ -43,6 +48,30 @@ Testing
 
 After programming the sample to your development kit, complete the following steps to test it:
 
+.. tabs::
```

Comment:
Just a note about the building and running: let's keep the current include as a placeholder, till it is more clear what's gonna be the build process for 54h in sdk-nrf. However, let's not merge the PR before this is clarified.